.github/workflows/make_release_concrete_tfhe_versionable.yml

```diff
@@ -7,22 +7,78 @@ env:
   ACTION_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
 
 jobs:
+  package:
+    runs-on: ubuntu-latest
+    outputs:
+      hash: ${{ steps.hash.outputs.hash }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+        with:
+          fetch-depth: 0
+      - name: Prepare package
+        run: |
+          cargo package -p tfhe-versionable-derive
+          cargo package -p tfhe-versionable
```

Comment:
as discussed this might break as tfhe-versionable depends on tfhe-versionable-derive

Reply:
Nice catch, it does not work indeed.